Classes/CalcsTab.lua

```diff
@@ -500,8 +500,8 @@ function CalcsTabClass:CalculatePowerStat(selection, original, modified)
 		original = original.Minion
 		modified = modified.Minion
 	end
-	originalValue = original[selection.stat] or 0
-	modifiedValue = modified[selection.stat] or 0
+	local originalValue = original[selection.stat] or 0
```

Comment:
I noticed and fixed this mistake while working on this PR. I can revert it if the change is inappropriate.

Reply:
Could you please refactor this out into a separate PR? The rest is looking great!